Sources/GRPCNIOTransportCore/Client/Connection/Connection.swift

```diff
@@ -421,6 +421,14 @@ extension Connection {
 
     private let http2Stream: NIOAsyncChannel<RPCResponsePart, RPCRequestPart>
 
+    var peerInfo: String {
+      self.http2Stream.channel.getRemoteAddressInfo()
```

Comment:
I'm not sure whether the remote info is actually set on the HTTP/2 stream channel. 

Also given this information won't change during the lifetime of a single connection, we can get both of these values up front when we create the HTTP/2 connection. That way we can avoid recreating the strings for each stream (which is potentially expensive).